core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -310,6 +310,9 @@ public OutputStream create(String path) throws IOException {
    * @throws IOException if a non-Alluxio error occurs
    */
   public OutputStream create(String path, CreateOptions options) throws IOException {
+    if (!options.isEnsureAtomic()) {
```

Comment:
Could we rename this to something like "writeDirect" or "direct"? The negation of "ensureAtomic" is "don't ensure anything", but the journal needs to ensure that when it flushes data that data is persisted in the file being written.